### PR TITLE
added a skill to index.md

### PR DESCRIPTION
**Pull request by AntonOlsson2**

### CODE REVIEW

File: index.md [hunk 1]
```diff
@@ -7,4 +7,4 @@ if you read this text uncommented text causes an error
 
 ## My skills
 
-* Git and GitHub
+* Git and GitHub and C+-
```
Comment (gusah153) [inline index.md:10]: Git and GitHub and LISP

File: index.md [hunk 2]
```diff
@@ -7,4 +7,4 @@ if you read this text uncommented text causes an error
 
 ## My skills
```
Comment (gusah153) [inline index.md:8]: Hello there

File: index.md (hunk 1 above)
Comment (AntonOlsson2) [reply index.md:10]: Maybe both C+- and LISP?

File: index.md (hunk 2 above)
Comment (AntonOlsson2) [reply index.md:8]: General Kenobi

File: index.md (hunk 1 above)
Comment (gusah153) [reply index.md:10]: Git and GitHub and C+- and LISP

**Review by gusah153 — CHANGES_REQUESTED** [review]

**Pull request merged**